courses.html

```diff
@@ -0,0 +1,186 @@
+                            <h3 class="para-title">Animation</h3>
+                            <p class=para-copy>Develop creative and technical abilities in character animation, visual storytelling, compositing and editing on this degree course and go into jobs in the animation field..</p>
+                        </div>
+                        <div class="buy-card-body">
+                            <div class="buy-body-row">
+                                <div class="buy-page-text text-hardness-low">
+                                    <p class="price-text">first year</p>
+                                </div>
+                                <div class="price price-right-align">€700.00
+                                </div>
+                            </div>
+                            <div class="buy-body-second-row">
+                                <div class="buy-page-text text-hardness-low">
+                                    <P>second year</P>
+                                </div>
+                                <div class="price-right-align">€600.00
+                                </div>
+                            </div>
+                             <div class="buy-body-third-row">
+                                <div class="buy-page-text text-hardness-low">
+                                    <a class="nav-links__link nav-links__link--color-red" href="index.html">Request demo</a>
+                                </div>
+                                <div class="price-right-align">
+                                    <a href="inndex.html" class="hero-btn hero-btn--red hero-btn--rounded">Buy</a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+            </div>
+        </section>
+        <section class="footer">
+            <h4 class="footer__heading">About Us</h4>
+            <p class="footer__copy">Excellence University is a leading Educational institution based out of Delhi. The University has established itself as a high quality education provider<br> with prime focus on holistic learning and imbibing competitive abilities in students.</p>
+            <div class="footer_icons">
+                <i class="fab fa-facebook style-icon"></i>
+                <i class="fab fa-twitter style-icon"></i>
+                <i class="fab fa-instagram style-icon"></i>
+                <i class="fab fa-linkedin style-icon"></i>
+                    
+            </div>
+        </section>  
+        
+         <script>
+        const allMatchingElementsText= document.querySelectorAll(".buy-page-text .price-text");
+        const allMatchingElementsPrice= document.querySelectorAll(".price");
+        const secondYearPrice= document.querySelectorAll(".buy-body-second-row"); 
+        document.getElementById("plan-selection").addEventListener("click", changeContentAsPerSelectedPlan);
```

Comment:
event should happen on to-be-changed element click, not on the whole box. it introduces the bug when whole box change styles:

<img width="423" alt="Screenshot 2021-07-06 at 11 12 39" src="https://user-images.githubusercontent.com/20513779/124583591-1e779900-de4b-11eb-8b1e-c8b1db898f11.png">